modules/virt-supported-features-for-virtual-storage-matrix.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-supported-features-for-virtual-storage.adoc
+
+[id="virt-supported-features-for-virtual-storage-matrix_{context}"]
+= Supported features for {VirtProductName} virtual storage
+
+.{VirtProductName} virtual storage feature support matrix
+[cols="40%,15%,15%,15%,15%"]
+|===
+||Virtual machine live migration  |Host-assisted virtual machine disk cloning |Storage-assisted virtual machine disk cloning |Virtual machine import
+
+|{product-title} container storage: RBD block volumes
+|Supported
+|Supported
+|Supported
+|Supported
+
+|{product-title} container storage: RBD filesystem volumes
```

Comment:
RBD is block device. you should state cephfs I guess

Reply:
@aglitke Good to know the clear description. I prefer we say like "Block mode PVC on a Ceph RBD device".

Reply:
No change per @aglitke comment.